test/unit/transaction-runner-test.coffee

```diff
@@ -327,20 +328,63 @@ describe 'TransactionRunner', ()->
           done()
 
     describe 'when a test has been manually set to skip in a hook', () ->
+      clonedTransaction = null
 
-      beforeEach () ->
+      beforeEach (done) ->
         sinon.stub configuration.emitter, 'emit'
+
+        clonedTransaction = clone(transaction)
```

Comment:
again, the same problem with cloned and origin. In first batch of `beforeEach` (in the `beforeHooks`) you just set the skip to true.

But - you've changed the transaction [origin] in that.
`clonedTransaction` is mutated again and again.